src/sentry/static/sentry/app/components/events/interfaces/togglableAddress.tsx

```diff
@@ -63,7 +63,7 @@ class TogglableAddress extends React.Component<Props> {
     const formattedAddress = !relativeAddress || isAbsolute ? address : relativeAddress;
 
     return (
-      <Address>
+      <Address display={!!formattedAddress}>
```

Comment:
@jan-auer thoughts on just always hiding the address on mobile?

Reply:
Hmm, it’s the least relevant of all the values for sure. But I think we don’t hide information usually on mobile view ports. Especially since it also contains the stack scan info. 

Maybe move it up next to the package?

Reply:
Not sure how much work that is. @priscilawebdev --^

Reply:
I am hiding this when there is no value; therefore, we do not display any empty space among other information. I couldn't just not render it, otherwise, I would break it on bigger screens